linux_os/guide/system/apparmor/ensure_apparmor_enforce_or_complain/oval/shared.xml

```diff
@@ -0,0 +1,32 @@
+<def-group>
+  <definition class="compliance" id="ensure_apparmor_enforce_or_complain" version="1">
```

Comment:
I  wanted to double-check that you wanted this rule to allow for both, even on level 2.

Reply:
hmm good catch, for level 2 I want it to be in enforce and to fail if in complain mode.
is there a good way to do this without creating two different rules?

Reply:
You should be able to check if the state matches a variable like in [selinux_state](https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/selinux/selinux_state/oval/shared.xml) rule.